src/locale/locales/es/messages.po

```diff
@@ -3755,64 +3755,64 @@ msgstr ""
 
 #: src/view/com/notifications/FeedItem.tsx:211
 msgid "liked your custom feed"
-msgstr ""
+msgstr "le gusta tu feed personalizado"
 
 #: src/view/com/notifications/FeedItem.tsx:178
 msgid "liked your post"
-msgstr ""
+msgstr "le gusta tu post"
 
 #: src/view/screens/Profile.tsx:222
 msgid "Likes"
 msgstr "Cantidad de «Me gusta»"
 
 #: src/view/com/post-thread/PostThreadItem.tsx:204
 msgid "Likes on this post"
-msgstr ""
+msgstr "Cantidad de  «Me gusta» en este post"
 
 #: src/Navigation.tsx:192
 msgid "List"
-msgstr ""
+msgstr "Lista"
 
 #: src/view/com/modals/CreateOrEditList.tsx:241
 msgid "List Avatar"
-msgstr "Avatar de la lista"
+msgstr "Lista de avatares"
```

Comment:
```suggestion
msgstr "Avatar"
```

The original string was correct, it's referring to the avatar/profile picture associated with the list